Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -96,7 +96,11 @@ GCPad::GCPad(const unsigned int index) : m_index(index)
   m_options->AddSetting(&m_always_connected_setting,
                         // i18n: Treat a controller as always being connected regardless of what
                         // devices the user actually has plugged in
-                        _trans("Always Connected"), false);
+                        {_trans("Always Connected"), _trans(""),
+                         _trans("Always connected if true.\nIt will link the emulated controller "
+                                "connetion state to the\n"
+                                "real default device connetion state if false (if there is one).")},
```

Comment:
Typo, 'connetion' -> 'connection'. Two instances.

Reply:
Also rereading that this is not a good comment to expose to a non-programmer user like that... Maybe replace the 'true' and 'false' with 'checked' and 'unchecked'?